src/logic/operate.js

```diff
@@ -0,0 +1,36 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const A = Big(Number(numberOne));
+  const B = Big(Number(numberTwo));
+  let C = 0;
```

Comment:
### [Optional]
Hello Dev, good work so far!
Let's practice a good naming convention.
we could use `firstNumber`, `secondNumber`, and `result` respectively instead of A, B, and C.

Reply:
Thanks. for the suggestion.
Implemented.
https://github.com/IvanDerlich/Calculator/pull/4/commits/304dac2620c23f8e22e406da4f3b892ce6bb1d98